frontend/javascripts/admin/team/edit_team_modal_view.tsx

```diff
@@ -0,0 +1,145 @@
+import { MinusOutlined, PlusOutlined } from "@ant-design/icons";
+import { getEditableUsers, updateUser } from "admin/admin_rest_api";
+import { Modal, AutoComplete, Input, Spin } from "antd";
+import { DefaultOptionType } from "antd/lib/select";
+import * as React from "react";
+import { useEffect, useState } from "react";
+import { APITeam, APITeamMembership, APIUser } from "types/api_flow_types";
+import { filterTeamMembersOf, renderUsersForTeam } from "./team_list_view";
+
+type Props = {
+  onCancel: (...args: Array<any>) => any;
+  isOpen: boolean;
+  team: APITeam | null;
+};
+
+function EditTeamModalForm({ onCancel, isOpen, team }: Props) {
+  const [autoCompleteValue, setAutoCompleteValue] = useState("");
+  const onChange = (newValue: string) => setAutoCompleteValue(newValue);
+  const [users, setUsers] = useState<APIUser[] | null>(null);
+  const fetchUsers = async () => setTimeout(async () => setUsers(await getEditableUsers()), 5000); // TODO delete; only testing purposes
+  //const fetchUsers = async () => setUsers(await getEditableUsers());
+  useEffect(() => {
+    fetchUsers();
+  }, []);
+
+  if (team === null) return null;
+  const updateTeamMembership = async (user: APIUser, newTeams: APITeamMembership[]) => {
+    if (users === null) return;
+    const newUser = Object.assign({}, user, {
+      teams: newTeams,
+    });
+    const serverUser = await updateUser(newUser);
+    setUsers(users.map((oldUser) => (oldUser.id === serverUser.id ? serverUser : oldUser)));
+  };
+
+  const addTo = async (user: APIUser, team: APITeam | null) => {
+    if (team === null) return;
+    const newTeam: APITeamMembership = { id: team.id, name: team.name, isTeamManager: false }; //TODO never make manager?
```

Comment:
:dodo: dont forget to delete

Reply:
Yes, I think, it's fair to add users as "normal" users without special permissions to a team. The user can use the users page to edit the roles for now.